modules/find.xq

```diff
@@ -47,14 +47,19 @@ declare variable $versions-or-version-range :=
         or exists($semver-max)
 ;
 
-(:
- input: "text/html, application/xhtml+xml, application/xml;q=0.9, image/webp, */*;q=0.8"
- output: ("text/html", "application/json", "application/xml", "image/webp", "*/*")
-:)
-declare function local:parse-accept-header($accept as xs:string) as xs:string* {
-    tokenize($accept, ",")
-        ! normalize-space() (: trim value :)
-        ! tokenize(., ";")[1] (: drop q :)
+(:~
+ : Read and split accept header into a list of mime types, if present
+ : q-values are ignored, order is preserved
+ :
+ : input: "text/html, application/xhtml+xml, application/xml;q=0.9, image/webp, */*;q=0.8"
+ : output: ("text/html", "application/json", "application/xml", "image/webp", "*/*")
+ :)
+declare function local:parse-accept-header() as xs:string* {
```

Comment:
Does it make sense to keep `as xs:string*` it we expect empty sequences as return?

Reply:
Zero or more